src/EditorFeatures/CSharpTest/ChangeSignature/ChangeSignature_Delegates.cs

```diff
@@ -18,7 +18,7 @@ namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.ChangeSignature
     [Trait(Traits.Feature, Traits.Features.ChangeSignature)]
     public partial class ChangeSignatureTests : AbstractChangeSignatureTests
     {
-        protected override CodeRefactoringProvider CreateCodeRefactoringProvider(Workspace workspace, TestParameters parameters)
+        protected override CodeRefactoringProvider CreateCodeRefactoringProvider(EditorTestWorkspace workspace, TestParameters parameters)
```

Comment:
Some overrides expect it to be EditorTestWorkspace (downcasting it), hiding dependency on editor features.

Reply:
This change was the source of the majority of unnecessary changes in this diff, and resulted in significant unnecessary test coupling to benefit a tiny (basically insignificant) set of test cases. I believe it will heavily benefit this change to restore the original signature.